dev-python/openpyxl/openpyxl-2.6.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
Can you test it with PyPy and PyPy3? If not, I'll test it when I merge the PR.

Reply:
`dev-python/et_xmlfile` doesn't support `PyPy` and `PyPy3`.
`dev-python/lxml` doesn't support `PyPy3`.
`dev-python/pillow` doesn't support `PyPy3`.

Reply:
My bad, I thought that I had checked all deps!